yarn.lock

```diff
@@ -2,310 +2,312 @@
 # yarn lockfile v1
```

Comment:
Why did this change?

Reply:
Don't the `yarn.lock` file change automatically? 😕

Reply:
Not if the dependencies don't change.